src/rawtoaces_util/usage_timer.cpp

```diff
@@ -23,40 +23,34 @@ void UsageTimer::reset()
     if ( enabled )
     {
 #ifndef WIN32
-        struct timeval start_timeval;
-        gettimeofday( &start_timeval, NULL );
-        _start_time = (double)start_timeval.tv_sec * 1000.0 +
-                      (double)start_timeval.tv_usec / 1000.0;
+        gettimeofday( &_start_timeval, NULL );
 #else
-        LARGE_INTEGER unit, start_timeval;
-        QueryPerformanceCounter( &start_timeval );
-        QueryPerformanceFrequency( &unit );
-        _start_time =
-            (double)start_timeval.QuadPart * 1000.0 / (double)unit.QuadPart;
```

Comment:
instead of calculating ms twice it is done once in print() with raw value stored.